sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -859,6 +998,11 @@ Language=Polish
 Menedżer sterowania usługami nie został zainicjowany pomyślnie. Nie udało się zainicjować serwera
 konfiguracji zabezpieczeń (scesrv.dll) z powodu błędu %1. Trwa ponowne uruchamianie systemu...
 .
+Language=Romanian
+Administratorul de control al serviciului s-a putut inițializa cu succes. Securitatea
+servitorului de configurație (scesrv.dll) s-a putut inițializa cu eroarea %1.
```

Comment:
Also, at the moment of writing this comment, I see:
`Administratorul de control al serviciului s-a putut inițializa cu succes. Securitatea
servitorului de configurație (scesrv.dll) s-a putut inițializa cu eroarea %1.`
...which reports something positive, whereas the correct message should report a failure instead.